threatstream.json

```diff
@@ -5,7 +5,7 @@
     "publisher": "Splunk",
     "type": "threat intel",
     "main_module": "threatstream_connector.py",
-    "app_version": "3.5.0",
+    "app_version": "3.5.1",
```

Comment:
If we don't have to do it I wouldn't do it, also this is more than just a patch if we were to split hairs